src/moment/formatDateOpened.js

```diff
@@ -0,0 +1,4 @@
+import moment from "moment/moment";
+
+
+export const formatDateOpened = (date) => moment(date).format("MMM Do YY"); 
```

Comment:
Nice! You could give it a more generic name though.

Reply:
Good idea!